checks/runchecks.go

```diff
@@ -11,15 +12,56 @@ import (
 	"github.com/flanksource/commons/logger"
 )
 
-func RunChecks(ctx *context.Context) []*pkg.CheckResult {
+// A list of check types that are permanently disabled.
+var disabledChecks map[string]struct{}
+
+func getDisabledChecks() (map[string]struct{}, error) {
+	if disabledChecks != nil {
+		return disabledChecks, nil
+	}
+
+	rows, err := db.Gorm.Raw("SELECT name FROM disabled_checks").Rows()
```

Comment:
This will throw an error for `canary-checker run` - Could we add an e2e test so that it picks it up in future ?